x/liquidity/spec/03_state_transitions.md

```diff
@@ -1,174 +1,235 @@
-<!--
-order: 3
--->
+<!-- order: 3 -->
 
-# State Transitions
+ # State Transitions
+
+Messages (Msg) trigger state transitions.
 
 ## Coin Escrow for Liquidity Module Messages
 
-Three messages on the liquidity module need prior coin escrow before confirmation, which causes state transition on `Bank` module. Below lists are describing coin escrow processes for each given message type.
+Transaction confirmation causes state transition on `Bank` module. Some messages on the Liquidity Module require coin escrow before confirmation.
+
+The coin escrow processes for each message type are:
```

Comment:
we are listing messages here, should we reference these state transitions from the messages doc?

Reply:
I think this `state_transition.md` needs a lot of room for improvement. It needs to cover batch-style execution and maybe all different types of message states that are operated by transaction messages.